### ISSUE TITLE
Add support for saving a game

### ISSUE THREAD

**Issue by RobertN** [sirjeppe/WebGameMaker#18]
Since we already seem to have a mechanism for "loading" games, we need to implement some way of storing them as well. IIRC we discussed storing them as JSON objects somehow, probably using local storage initially?

**Comment by sirjeppe** [comment]
Yeah! Good idea... Should it be a dialog that is blocking the inital loading? Like... Cubase etc. is doing? Showing this initial dialog before loading the rest of the UI basically. Maybe you will want to have different workspace setup (window positions etc.) for different projects?
